.github/workflows/build_and_test.yaml

```diff
@@ -122,8 +122,8 @@ jobs:
         - version: v1.30.4
           ipFamily: ipv4
         # Enable these after https://github.com/envoyproxy/gateway/issues/4572 fixed
-        # - version: v1.31.0
-        #   ipFamily: ipv6   # only run ipv6 test on latest version to save time
+        - version: v1.31.0
```

Comment:
can we do the for v1.30 so we save some time ?

Reply:
sure